framework/source/class/qx/util/ResourceManager.js

```diff
@@ -291,8 +298,16 @@ qx.Class.define("qx.util.ResourceManager",
           }
           // If the resourceUri begins with a single slash, include the current
           // hostname
-          else if (resourceUri.match(/^\//) != null) {
-            statics.__urlPrefix[lib] = window.location.protocol + "//" + window.location.host;
+          else if (resourceUri.match(/^\//) != null)
```

Comment:
To match with the existing code style, the braces should not start on a new line.